src/build.props

```diff
@@ -42,7 +42,7 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="'$(DisableBinSkimTarget)'!='true'"/>
   <PropertyGroup>
     <BinSkimVerboseOutput>false</BinSkimVerboseOutput>
-    <BinSkimExePath>$(MsBuildThisFileDirectory)packages\Microsoft.CodeAnalysis.BinSkim.1.3.9\tools\x64\BinSkim.exe</BinSkimExePath>
+    <BinSkimExePath>$(MsBuildThisFileDirectory)packages\Microsoft.CodeAnalysis.BinSkim.1.4.0\tools\x64\BinSkim.exe</BinSkimExePath>
   </PropertyGroup>
```

Comment:
SARIF SDK runs binskim as part of its production build!

---
In reply to: [117521614](https://github.com/Microsoft/sarif-sdk/pull/570#discussion_r117521614) [](ancestors = 117521614)